modules/core/arrow-core-data/src/test/kotlin/arrow/typeclasses/BifoldableTests.kt

```diff
@@ -32,6 +35,10 @@ class BifoldableTests : UnitSpec() {
 
     val eitherComposeEither = eitherBifoldable.compose(eitherBifoldable)
 
-    testLaws(BifoldableLaws.laws(eitherComposeEither, { cf: Int -> Right(Right(cf)).binest() }, Eq.any()))
+    val g = Gen.intSmall().map {
+      Right(Right(it)).binest()
+    } as Gen<Kind<Kind<Nested<ForEither, ForEither>, Int>, Int>>
```

Comment:
This is not  a good gen. A nested either gen should be able to gen all options

Reply:
ok, now using a nested gen